core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.scss

```diff
@@ -0,0 +1,80 @@
+@use "variables" as *;
+
+::ng-deep {
+    .p-card {
+        border: 1px solid $color-palette-primary-300;
+    }
+
+    .p-card .p-card-body {
+        padding: 0;
+    }
+
+    .template-builder-box {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    .template-builder-box-item {
+        padding: 0px $spacing-1 0px $spacing-3;
+        border: 1px solid $color-palette-primary-300;
+        margin: $spacing-1 $spacing-3;
+
+        &:first-child {
+            margin-top: $spacing-3;
+        }
+
+        &:last-child {
+            margin-bottom: $spacing-3;
+        }
+
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        p {
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+        }
```

Comment:
On top of this file you can do `@import "mixins";` and then here do `@include truncate-text;` that way we use the truncate mixin created on global variables.